kola/harness.go

```diff
@@ -192,6 +192,11 @@ func filterTests(tests map[string]*register.Test, pattern, platform string, vers
 			continue
 		}
 
+		if platform == "qemu" && t.HasFlag(register.RequiresInternetAccess) {
```

Comment:
Generally I'd rather have a list of platforms that do not have the feature and then querying if the platform is in the list. However, I don't think we'll be adding any new platforms that don't have internet access so I don't think it will matter.

Reply:
Thanks, SGTM to have it a list. Will be clearer with it being named something like `platformsNoInternet`. Will make that edit in